### DISCUSSION TITLE
how does the doh dns resolver option function?

### DISCUSSION THREAD

**Discussion by advait404** [winsiderss/systeminformer#1950] [General Discussion]
Hi, I see that there is a doh dns resolver option in the settings. how does this work exactly?

**Comment by dmex** [comment]
Standards are documented by providers: https://developers.cloudflare.com/1.1.1.1/encryption/dns-over-https/make-api-requests/

**Reply by advait404** [reply]
Thanks, I do know about DOH and how it works. I was curious about what system informer is doing related to that. Is it a DNS host resolver similar to the NextDNS windows client, or does it do something else?

**Reply by dmex** [reply]
Sorry for the delayed response, Github never sent a notification.

System Informer is using DoH for displaying hostnames on the network tab that are consistent with the hostname resolution of browsers and for accuracy of results without anyone tampering with responses.
